dev-php/awl/awl-0.59.ebuild

```diff
@@ -7,7 +7,7 @@ DESCRIPTION="Andrew McMillan's Web Libraries"
 HOMEPAGE="https://gitlab.com/davical-project/awl"
 SRC_URI="https://www.davical.org/downloads/${PN}_${PV}.orig.tar.xz -> ${P}.tar.xz"
 
-LICENSE="GPL-2"
+LICENSE="GPL-2 GPL-2+ GPL-3+ LGPL-2+ LGPL-3+"
```

Comment:
Preferably, the license update goes into its own commit :) I am fine with this if you dont want to fix it, but...

Reply:
no problem, will do so